seqr/views/utils/dataset_utils.py

```diff
@@ -377,7 +387,8 @@ def _load_rna_seq(model_cls, file_path, user, mapping_file, ignore_extra_samples
         row = dict(zip(header, _parse_tsv_row(line)))
         for sample_id, row_dict in parse_row(row, sample_id_to_tissue_type=sample_id_to_tissue_type):
             gene_id = row_dict['gene_id']
-            existing_data = samples_by_id[sample_id].get(gene_id)
+            project = row_dict.pop(PROJECT_COL, None)
```

Comment:
We will remove `None` after all datasets have been upgraded to providing a project column.